recipes-bsp/host-watchdog-fw/host-watchdog-fw_1.0.bb

```diff
@@ -0,0 +1,40 @@
+SUMMARY = "rpmsg lite host watchdog firmware"
+DESCRIPTION = "Cortex M4/M7 Firmware ${SUMMARY} for Host Monitor X"
```

Comment:
Are you sure this works for M4?

Reply:
if it has the correct board, pinmux and clock configuration files for m4 it should work.